samples/alloy-editor/README.md

```diff
@@ -13,14 +13,13 @@ Install dependencies if not done already through `npm install`.
 
 To use the extension from your local environment:
 * Fill in your space id in the `Makefile`.
-* Run `make create` to upload your extension to the server
-* Run `make serve` to serve the extension
+* Run `make serve` to serve the extension from port 3000.
```

Comment:
It should be

- `make create`
- `make update-local`
- `make serve`

Reply:
I see. I've looked through these instructions with two other engineers (for various reasons) and we were all initially confused with the installation instructions because not everyone wants or needs to develop this extension locally and can upload straight to Contentful. 

Not a big deal (especially since people can now click a button to upload extensions in the web app :smile:) and we can probably close this PR.

Reply:
yeah I agree - the installation instructions are misleading and we plan to consolidate all of them to follow the same approach.

Thanks for the feedback @christineywang